packages/server/medplum.config.json

```diff
@@ -34,8 +34,18 @@
     "password": "medplum"
   },
   "bullmq": {
-    "removeOnFail": { "count": 1 },
-    "removeOnComplete": { "count": 1 }
+    "removeOnFail": {
+      "count": 1
+    },
+    "removeOnComplete": {
+      "count": 1
+    }
   },
-  "shutdownTimeoutMilliseconds": 30000
+  "shutdownTimeoutMilliseconds": 30000,
+  "externalAuthProviders": [
+    {
+      "issuer": "http://127.0.0.1:4444",
+      "userInfoUrl": "http://127.0.0.1:4444/userinfo"
+    }
+  ]
```

Comment:
This config shouldn't make it's way into main, but is useful for the external auth POC